src/components/tests/dropdown.component.test.js

```diff
@@ -64,4 +65,96 @@ describe('[DropdownComponent]', () => {
             expect(dropdown.list.attr('style')).toEqual('display: none;');
         });
     });
+
+    describe('[setSelection]', () => {

```

Comment:
What happens if i try to select a item that does not exist in the dropdown?

Reply:
Now it's throw new Error('Item not find!')